includes/tutorials-editors-customize-drop-down-menu.md

```diff
@@ -0,0 +1,74 @@
+The {WidgetName} uses the [Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/) component as the drop-down menu. To customize the menu, specify [Popup properties](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/) in the [dropDownOptions](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#dropDownOptions) object. For example, the following code sets the height of {WidgetName}'s drop-down menu to ${{value}} pixels:
```

Comment:
The widget uses the Popup component as a drop-down menu in general. See above.
```suggestion
The {WidgetName} uses the [Popup](/Documentation/Guide/UI_Components/Popup/Getting_Started_with_Popup/) component as a drop-down menu. To customize the menu, specify [Popup properties](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/) in the [dropDownOptions](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#dropDownOptions) object. For example, the following code sets the height of {WidgetName}'s drop-down menu to ${{value}} pixels:
```

Reply:
Same as above

Reply:
Either way, I told you my opinion (see the identical change above), do with it what you will. Your word against mine.